src/Sylius/Bundle/OrderBundle/Command/RemoveExpiredCartsCommand.php

```diff
@@ -27,7 +27,7 @@ protected function configure()
     {
         $this
             ->setName('sylius:remove-expired-carts')
-            ->setDescription('Removes carts that have been idle for a configured period. Configuration parameter - sylius_order.cart_expires_after.');
+            ->setDescription('Removes carts that have been idle for a configured period. Configuration parameter - sylius_order.cart_expiration_period.');
```

Comment:
Well I guess that's a matter of taste. I prefer to refer to the parameter as it is a much quicker way (1 line instead of 3) to set the same thing.

If we were to refer to the configuration structure we should also change CancelUnpaidOrdersCommand.